apps/next-docs/content/components/AnchorNav/index.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Anchor nav
+description: Use the anchor nav component for prompt navigation to different sections of a page.
+keywords: ['navigation', 'menu', 'sticky', 'links', 'sections']
+show-tabs: false
+ready: true
+figma: https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=3359%3A30494&t=7vT79cAF1kbiMd2j-4
+source: https://github.com/primer/brand/blob/main/packages/react/src/AnchorNav/AnchorNav.tsx
+storybook: '/brand/storybook/?path=/story/components-anchornav--playground'
+---
+
+import {Label} from '@primer/react'
+import {AnchorNavChildrenProp} from './react.tsx'
+
+```js
+import {AnchorNav} from '@primer/react-brand'
+```
+
+## Examples
+
+<Note>
+
+`AnchorNav` is designed to fix to the top of the viewport, and requires markup outside of the scope of the component to behave correctly.
+
+Please refer to our [Storybook examples](https://primer.style/brand/storybook/?path=/story/components-anchornav--playground) to see the component in a full-screen context as originally intended.
+
+</Note>
+
+### Basic
```

Comment:
This example feels quite broken. Any thoughts/ideas on what we should do to fix these type of live previews @danielguillan?
![Uploading Screenshot 2025-04-23 at 11.56.36.png…]()